src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -102,6 +102,23 @@ public virtual bool IsGenericMethod
 
         internal virtual ImmutableArray<string> NotNullWhenFalseMembers => ImmutableArray<string>.Empty;
 
+#nullable enable
+        internal abstract UnmanagedCallersOnlyAttributeData? UnmanagedCallersOnlyAttributeData { get; }
+
+        internal void ForceCompleteUnmanagedCallersOnlyAttribute()
+        {
+            if (ReferenceEquals(UnmanagedCallersOnlyAttributeData, UnmanagedCallersOnlyAttributeData.Uninitialized)
+                || ReferenceEquals(UnmanagedCallersOnlyAttributeData, UnmanagedCallersOnlyAttributeData.AttributePresentDataNotBound))
```

Comment:
>|| ReferenceEquals(UnmanagedCallersOnlyAttributeData, UnmanagedCallersOnlyAttributeData.AttributePresentDataNotBound) [](start = 16, length = 117)

Is this check needed? Won't the `AttrbuteData` be `Uninitialized` in this case?

Reply:
No. Early binding could have occurred but not full binding, meaning that the data could have not been fully initialized.

---
In reply to: [483714061](https://github.com/dotnet/roslyn/pull/47311#discussion_r483714061) [](ancestors = 483714061)